server/src/main/java/org/elasticsearch/action/get/TransportGetAction.java

```diff
@@ -116,7 +116,8 @@ protected GetResponse shardOperation(GetRequest request, ShardId shardId) throws
                 request.realtime(),
                 request.version(),
                 request.versionType(),
-                request.fetchSourceContext()
+                request.fetchSourceContext(),
+                request.isForceSyntheticSource()
```

Comment:
This was making me think that maybe we should add `forceSynthetic` as a member on FetchSourceContext, but it looks as though that is used in loads of other places so it would end up being a much bigger change.

Reply:
Yeah. I thought about it and didn't like.